app/logic/fileHandler.py

```diff
@@ -79,6 +79,7 @@ def saveFiles(self, saveOriginalFile=None):
     def retrievePath(self,files):
         pathDict={}
         for file in files:
+            
             pathDict[file.fileName] = ((self.path+"/"+ file.fileName)[3:], file)
```

Comment:
Please space evenly before and after the additions in your string concatenation.